service/src/main/java/org/mskcc/smile/service/impl/TempoServiceImpl.java

```diff
@@ -184,9 +193,6 @@ private void populateTempoData(SmileSample sample, Tempo tempo) throws Exception
                         ? ACCESS_LEVEL_PUBLIC : ACCESS_LEVEL_EMBARGO);
             }
         } else {
-            // explicitly set dates to empty strings if no initial pipeline run date is found
-            tempo.setInitialPipelineRunDate("");
-            tempo.setEmbargoDate("");
```

Comment:
neo4j will ignore nulls but not empty strings when evaluating `date(t.embargoDate)` 

if left as empty strings instead then neo4j/spring data throws an error when running the query to find tempo ids no longer under embargo since it tries to parse the empty string as a date